modules/cco-ccoctl-configuring.adoc

```diff
@@ -275,6 +275,42 @@ ifdef::azure-workload-id[]
 ====
 endif::azure-workload-id[]
 
+//GCP permissions needed when running ccoctl during install.
+ifdef::google-cloud-platform[]
+* You have created a GCP account for the `ccoctl` utility to use with the **IAM Workload Identity Pool Admin** role and the following permissions:
```

Comment:
Similar to **IAM Role Administrator** above, the **IAM Workload Identity Pool Admin** role would be added to the installer service account. It includes many of the permissions listed below. So the user would either use the installer account with IAM Workload Pool Admin, OR they would use the following absolute list of permissions.